CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
 # Chai Postman Changelog
 
+#### v0.1.1 [Unreleased] (November 4, 2017)
```

Comment:
Unreleased versions should not have dates.

Reply:
I think this is a breaking change. Can we make this `0.2.0` at least?

`1.0.0` would be best.

Reply:
@kunagpal - yes - remove this 0.0.1

```
#### Unreleased
- ...
- ...
- ...
```